tools/gitstatistics.py

```diff
@@ -316,7 +316,7 @@ def fetch_recent_activity(self, weeks=None):
             if weeks is None:
                 activity[yyw] = activity.get(yyw, 0) + 1
             elif yyw in weeks:
-                activity[yyw] = activity.get(yyw, 0) + 1
+                actstyleivity[yyw] = activity.get(yyw, 0) + 1
```

Comment:
misprint?

Reply:
Wow... soo accurate. Nice :). 
I fixed and thanks